acceptance/ui/features/hosts.feature

```diff
@@ -18,15 +17,204 @@ Feature: Host Management
       And I should see "Showing 0 Results"
 
   @login-required
-  Scenario: Add a host
+  Scenario: View Add Host dialog
     When I am on the hosts page
       And I click the Add-Host button
-    Then I should see "Host and port"
+    Then I should see the Add Host dialog
+      And I should see "Host and port"
+      And I should see the Host and port field
       And I should see "Resource Pool ID"
-    When I fill in the Host Name field with "gjones-dev:4979"
+      And I should see the Resource Pool ID field
+      And I should see "RAM Commitment"
+      And I should see the RAM Commitment field
+
+  @login-required @emptyHostsPage
+  Scenario: Add an invalid host with an invalid name
+    When I am on the hosts page
+      And I click the Add-Host button
+      And I fill in the Host Name field with "bogushost"
+      And I fill in the Resource Pool field with the default resource pool
+      And I fill in the RAM Commitment field with the default RAM commitment
+      And I click "Add Host"
+    Then I should see "Error"
+      And I should see "Bad Request"
+      And I should see "No Data Found"
```

Comment:
Do you think the test would be more readable if instead of checking for "No Data Found" and "Showing 0 Results", we had a step like `And I should see an empty Hosts page`